lib/ui/src/components/MDXRenderer/MDXRenderer.tsx

```diff
@@ -1,31 +1,45 @@
-import { MDXRemote, MDXRemoteProps } from "next-mdx-remote";
 import {
+  Box,
+  Divider,
   Heading,
+  Image,
+  Link,
+  ListItem,
+  OrderedList,
   Text,
   UnorderedList,
-  OrderedList,
-  ListItem,
-  Link,
-  Box,
   chakra,
-  Divider,
-  Image,
 } from "@chakra-ui/react";
-import NextLink from "next/link";
-import Head from "next/head";
 import { MDXProvider as BaseMDXProvider } from "@mdx-js/react";
 import { kebabCase } from "lodash-es";
-import { Terminal } from "../../components/Terminal/Terminal";
+import { MDXRemote, MDXRemoteProps } from "next-mdx-remote";
+import Head from "next/head";
+import NextLink from "next/link";
+import { ComponentProps, ReactNode, useCallback, useState } from "react";
 import { FAQItem } from "../../components/FAQItem/FAQItem";
-import { ReactNode, ComponentProps, useState, useCallback } from "react";
+import { Terminal } from "../../components/Terminal/Terminal";
 import { useSmoothScrollToHash } from "../../hooks/useSmoothScrollToHash";
-import { createBreakpointArray } from "../../theme/components/breakpoints";
 import { BlockQuote } from "../BlockQuote/BlockQuote";
 
 export function headingToAnchorId(headingEl: HTMLHeadingElement) {
   return kebabCase(headingEl.innerText.toLowerCase());
 }
 
+function Warning(props: ComponentProps<typeof Box>) {
+  return (
+    <Box
+      textAlign={"center"}
+      as="aside"
```

Comment:
Nitpicking, but I'd probably prefer to change this to a plain ol div. I think these specific html5 elements invite bikeshedding over whether they're appropriate for the use -- in this case, according to MDN https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside, it seems like asides are only for content that isn't related to the document, and I'd say the content is critical to the document.

Reply:
I tried to follow the pattern used everywhere else in this file. I agree with you that being accurate with the HTML semantics is the best. I'll convert it to a div and push

Reply:
Removed the "aside" field. The box component gets converted to a div: 

<img width="1512" alt="image" src="https://github.com/iron-fish/website/assets/13268167/ac87e157-c891-465c-8054-9f629b0eeb9a">

Do you still want me to change it to a div? I did this so that it is consistent with the rest of the repo? We actually don't have any plain `div` usages anywhere: 

<img width="612" alt="image" src="https://github.com/iron-fish/website/assets/13268167/72392056-de5f-4614-98e0-def5d2698b2b">